### PR TITLE
Why better update and footer date

### PR DESCRIPTION
**Pull request by klmhsb42**
Due to several "why better" pull requests, the oldest was published to website. You can update with this. It includes images, too.

### CODE REVIEW

File: why.html [hunk 1]
```diff
@@ -119,14 +100,14 @@ <h1 data-i18n="why_use_syncloud">Why Use Syncloud?</h1>
 			<div class="tablecel"><div class="tic" data-i18n="data_control">Full Data Control</div><div class="negative" data-i18n="no">No</div></div>
 			<div class="tablecel"><div class="tic" data-i18n="data_protection">Data Protection (Backup possible)</div><div class="positive" data-i18n="yes">Yes</div></div>
 			<div class="tablecel"><div class="tic" data-i18n="open_source_software">Open source software (only)</div><div class="negative" data-i18n="no">No</div></div>
-			<div class="tablecel"><div class="tic" data-i18n="decentralized">Decentralized</div><div class="negative" data-i18n="no">No</div></div>
+			<div class="tablecel"><div class="tic" data-i18n="decentralized">Decentralized</div><div class="positive" data-i18n="yes">Yes</div></div>
 			<div class="tablecel"><div class="tic" data-i18n="easy_to_use">Easy-to-use</div><div class="positive" data-i18n="yes">Yes</div></div>
```
Comment (cyberb) [inline why.html:104]: Is this true for non-technical users?

Comment (klmhsb42) [reply why.html:104]: Are you asking in general or for Synology?

Comment (cyberb) [reply why.html:104]: Synology

Comment (klmhsb42) [reply why.html:104]: I've never used it, just saw a few videos about it. I guess it is easy to setup compared to other home server solutions? At least they are popular.

Comment (klmhsb42) [reply why.html:104]: Regarding the "Decentralized" I thought every home server is decentralized.

**Comment by cyberb** [comment]
Published to syncloud.org

**Comment by klmhsb42** [comment]
I found another mistake under "Full Data Control" for Syncloud should be yes (it's green), sorry for that

**Comment by klmhsb42** [comment]
> 
> 
> I found another mistake under "Full Data Control" for Syncloud should be yes (it's green), sorry for that

Probably wrong in the JSON

**Comment by cyberb** [comment]
Could you create a pull request for that?

**Comment by klmhsb42** [comment]
Yes, I will do that, should be somewhere lost in my old branches...

**Pull request merged**